101-vm-simple-linux/azuredeploy.json

```diff
@@ -14,6 +14,13 @@
         "description": "Password for the Virtual Machine."
       }
     },
+    "vmSize": {
```

Comment:
We want to avoid this pattern of putting VM sizes in parameters - it creates a maintenance problem.  Let's pick a size that works for the scenario and leave it as a var.